src/pages/vaults/VaultDepositWithdrawForm.tsx

```diff
@@ -530,10 +534,19 @@ export const VaultDepositWithdrawForm = ({
       </AlertMessage>
     ));
 
+  const isAccountViewOnly = useAppSelector(calculateIsAccountViewOnly);
+  const canViewAccount = useAppSelector(calculateCanViewAccount);
+  const disableFormBecauseWallet = isAccountViewOnly || !canViewAccount;
```

Comment:
omega-nit: `disableFormBecauseWallet` -> something with a `is`, `should`, `has`, `can` prefix. i.e. `isDisabledByAccountState`